ravenframework/ROMExternal.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on October 12, 2022
+
+@author: cogljj
+
+ROM Loader for serialized surrogate model (ROM) for internal usage
+Unlike externalROMloader.ravenROMexternal, this assumes ravenframework is
+already in the path.
+"""
+
+import os
+import pickle
+import numpy as np
+
+class ROMLoader(object):
+  """
+    External ROM loader and user-gate
+  """
+  def __init__(self, binaryFileName, _whereFrameworkIs=None):
+    """
+      This constructor un-serializes the ROM generated by RAVEN and
+      it makes the ROM available for external usage
+      @ In, binaryFileName, str, the location of the serialized (pickled) ROM that needs to be imported
+      @ In, _whereFrameworkIs, None, ignored
```

Comment:
this is to be consistent with the external ROM loader?

Reply:
Yes, that was the reason I left it. (Of course, in HERON, I immediately removed the second argument when I changed the import, so maybe it should just be removed.)